### PR TITLE
Add initial enviro+ support

### PR DESCRIPTION
**Pull request by shaunmulligan**
Add Enviro+ support

Currently only supports: Temp, Humidity, Pressure, Light (Lux), NH3, Oxidising and Reducing Gases

Connects-to: #35
Change-type: minor
Signed-off-by: Shaun Mulligan <shaun@balena.io>

### CODE REVIEW

**Comment by shaunmulligan** [comment]
Currently Temp, Humidity, Pressure, Light (Lux), NH3, Oxidising and Reducing Gases are supported. One of the problems is that it seems adding this new dashboard makes it the default, which isn't great, need to figure out how to set the default/home dashboard.

Next up need to add noise level and particulate sensor (with optional setting)

File: README.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ Here’s the shopping list for this project. Depending if you’d like to crack
 * Bosch BME680 sensor with breakout board (see below for places to find one) or...
 * **Optional:** Sense HAT (optional replacement for the BME680, but does not include an air quality sensor)
 * **Optional:** 1-wire temperature sensor (temperature only)
+* **Optional:** [Pimoroni Enviro+ Air Quality](https://shop.pimoroni.com/products/enviro?variant=31155658457171)
```
Comment (rahul-thakoor) [inline README.md]: perhaps we should note that only  supports Temp, Humidity, Pressure, Light (Lux), NH3, Oxidising and Reducing Gases for now?

Comment (shaunmulligan) [reply README.md]: @rahul-thakoor updated

**Comment by shaunmulligan** [comment]
@balena-ci retest

**Pull request merged**